deliverables/ruby_monk_problems/calculator.rb

```diff
@@ -1,9 +1,9 @@
-class Calculator
-  def add(a, b)
-   # your code here
+class Calculator  
+  def add(num_1, num_2)
```

Comment:
Looks like a copy of the solution.  I hope there is some understanding of what is happening in the code